src/test/java/bio/terra/service/resourcemanagement/google/BucketResourceTest.java

```diff
@@ -122,7 +123,9 @@ public void setup() throws Exception {
   @After
   public void teardown() throws Exception {
     for (GoogleBucketResource bucketResource : bucketResources) {
-      deleteBucket(bucketResource);
+      if (bucketResource != null) {
+        deleteBucket(bucketResource);
+      }
```

Comment:
I would push the null check into `deleteBucket()`, unless there's a case where we need `deleteBucket()` to throw when `null` is passed.
```
    bucketResources.forEach(BucketResourceTest::deleteBucket);
```